Diary/Diary.BusinessLogic/Models/JournalOwner/JournalOwnerFilterDto.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Diary.BusinessLogic.Models.DiaryOwner
+{
+    public class JournalOwnerFilterDto
+    {
+
+        [Required]
+        public Guid Id { get; init; }
+        [Required]
+        public required string Name { get; init; }
+        [Required]
+        public required string Email { get; init; }
+
```

Comment:
Why duplication "required"?

Reply:
just in case, using Required attribute from DataAnnotations is not required in that case when you are going to use FluentValidation at the beginning of handling request. It will be pointless to mix 2 approaches when you can use FluentValidation only